specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-10-02-preview/managedClusters.json

```diff
@@ -926,8 +926,20 @@
           }
         ],
         "responses": {
-          "204": {
-            "description": "NoContent"
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/AgentPool"
+            }
```

Comment:
> A 202 response should include an Location response header.

> The x-ms-long-running-operation-options should be specified explicitly to indicate the type of response header to track the async operation.

Maybe something like
```
        "x-ms-long-running-operation-options": {
          "final-state-via": "location"
        },
```

Reply:
> Do we really have agentpool/managedcluster returned in 202 response?
> 
> > Response statusCode 202 for operation ManagedClusters_AbortLatestOperation has no response body provided in the example, however the response does have a "schema" defined in the swagger spec.

Yes because in the RP code we write header and entity - header with the location URL and entity is the MC/AP content so that the users can see the provisioning state being updated. I added the body to the example.

Reply:
> > A 202 response should include an Location response header.
> 
> > The x-ms-long-running-operation-options should be specified explicitly to indicate the type of response header to track the async operation.
> 
> Maybe something like
> 
> ```
>         "x-ms-long-running-operation-options": {
>           "final-state-via": "location"
>         },
> ```

Added this